src/raw/mod.rs

```diff
@@ -43,14 +43,23 @@ use self::imp::Group;
 // consistently improves performance by 10-15%.
 #[cfg(feature = "nightly")]
 use core::intrinsics::{likely, unlikely};
+
+#[cfg(not(feature = "nightly"))]
```

Comment:
Can you add a comment here explaining that LLVM always treats calls to cold functions as unlikely?